pkgs/games/hase/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, pkg-config
+, stdenv
+, sparrow3d
+, zlib
+}:
+
+stdenv.mkDerivation {
+  pname = "hase";
+  version = "unstable-2020-10-06";
+
+  src = fetchFromGitHub {
+    owner = "theZiz";
+    repo = "hase";
+    rev = "31d6840cdf0c72fc459f10402dae7726096b2974";
+    hash = "sha256-d9So3E8nCQJ1/BdlwMkGbaFPT9mkX1VzlDGKp71ptEE=";
+  };
+  patches = [ ./prefer-dynamic.patch ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    sparrow3d
+    zlib
+  ];
+
+  buildPhase = ''
+    # hase expects its dependencies to already be on the ld path
+    NIX_CFLAGS_COMPILE=$(pkg-config --cflags sparrow3d zlib)
+
+    # build and install are one step, and inseparable without patching
+    mkdir -p $out/{bin,share/applications,share/pixmaps}
+    ./install.sh $out
+  '';
+
+  postFixup = ''
+    substituteInPlace "$out/share/applications/hase.desktop" \
+      --replace "Exec=hase" "Exec=$out/bin/hase"
+  '';
+
+  meta = with lib; {
+    description = "Hase is an open source gravity based artillery shooter. It is similar to Worms, Hedgewars or artillery, but the gravity force and direction depends on the mass nearby. It is optimized for mobile game consoles like the GP2X, Open Pandora or GCW Zero";
+    homepage = "http://ziz.gp2x.de/hase/";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ colinsane ];
+    platforms = platforms.linux;
```

Comment:
```suggestion
    license = lib.licenses.gpl3Plus;
    maintainers = with lib.maintainers; [ colinsane ];
    platforms = lib.platforms.linux;
```

Reply:
for the license (gpl3 -> gpl3Plus): i was going off <https://github.com/theZiz/hase/blob/master/LICENSE>.

it doesn't make any mention of gpl3Only v.s. gpl3Plus, and i'm not knowledgeable enough to know which one is appropriate. i'll apply this suggestion though -- i guess when a GPL3 license doesn't explicitly say otherwise, GPL3+ is assumed?